insights/parsers/tests/test_certificates_enddate.py

```diff
@@ -89,6 +96,9 @@ def test_certificates_enddate():
     assert (set(Cert5.certificates_path) == set([
             '/etc/pki/ca-trust/extracted/pem/email-ca-bundle.pem']))
 
+    Cert6 = CertificatesEnddate(context_wrap(CRT7))
+    assert not Cert6.data
```

Comment:
In this case, `SkipException` should be thrown, Would you mind improve the logic of this Parser?

Reply:
@xiangce, updated, please help to review again, thanks.